libkbfs/folder_block_ops.go

```diff
@@ -36,6 +36,9 @@ const (
 	blockRead blockReqType = iota
 	// A block write request.
 	blockWrite
+	// A block read request that is happening from a different
+	// goroutine than the blockLock rlock holder, using the same lState.
```

Comment:
Does that mean that if all n children goroutines use the same `lState`? If so, that seems incorrect, as an `lState` is meant to only apply for a single 'execution flow', which means a single goroutine or a serial sequence of goroutines.

Probably should enforce something like a nil lState exactly when using blockReadParallel.